packages/noports_core/lib/src/sshnp/impl/sshnp_openssh_local_impl.dart

```diff
@@ -94,6 +109,16 @@ class SshnpOpensshLocalImpl extends SshnpCore
       );
     }
 
+    /// Start sshrv
+    await sshrvdChannel.runSshrv(
+      directSsh: true,
+      localRvPort: localRvPort,
+      sessionAESKeyString: sshnpdChannel.sessionAESKeyString,
+      sessionIVString: sshnpdChannel.sessionIVString,
+    );
+
+    await Future.delayed(Duration(seconds: 1));
```

Comment:
Had to add this in order to give the sshrv process time to start up and bind to its port

Reply:
nvm... we would have to make that process fork and return an exit code, which we currently don't do.